File: BandBuilderJPA/src/main/java/com/skilldistillery/bandbuilder/entities/Address.java
package com.skilldistillery.bandbuilder.entities;

import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import org.hibernate.annotations.CreationTimestamp;
import org.hibernate.annotations.UpdateTimestamp;

@Entity
public class Address {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;

	private String street;

	@Column(name = "street_2")
	private String street2;

	private String city;

	private String state;

	private int zip;

	private Integer phone;

	@Temporal(TemporalType.TIMESTAMP)
	@CreationTimestamp
	@Column(name = "created_at")
	private Date createdAt;

	@Temporal(TemporalType.TIMESTAMP)
	@UpdateTimestamp
	@Column(name = "updated_at")
	private Date updatedAt;

	public Address() {
		super();
	}

}

File: ngBandBuilder/src/app/models/address.ts
export class Address {

id: number;
street: string;
street2: string;
city: string;
state: string;
zip: string;
phone: string;
createdAt: string;
updatedAt: string;
active: boolean;

constructor(
  id?: number,
  street?: string,
  street2?: string,
  city?: string,
  state?: string,
  zip?: string,
  phone?: string,
  createdAt?: string,
  updatedAt?: string,
  active?: boolean
) {
  this.id =  id;
  this.street =  street;
  this.street2 =  street2;
  this.city =  city;
  this.state =  state;
  this.zip =  zip;
  this.phone =  phone;
  this.createdAt =  createdAt;
  this.updatedAt =  updatedAt;
  this.active = active;
}


}
